roles/calico/templates/calico.service.j2

```diff
@@ -10,18 +10,22 @@ ExecStart=/usr/bin/docker run --net=host --privileged \
  --name=calico-node \
  -e WAIT_FOR_DATASTORE=true \
  -e FELIX_DEFAULTENDPOINTTOHOSTACTION=ACCEPT \
- -e CALICO_IPV4POOL_IPIP=always \
- -e FELIX_IPV6SUPPORT=false \
+ -e CALICO_IPV4POOL_IPIP={{ calico_ipv4pool_ipip }} \
+ {% if calico_ipv4pool_cidr is defined %}
+-e CALICO_IPV4POOL_CIDR={{ calico_ipv4pool_cidr }} \
+{% endif %}
+ -e FELIX_IPV6SUPPORT={{ felix_ipv6support }} \
  -e ETCD_ENDPOINTS={{ etcd_endpoints }} \
  -v /etc/origin/calico:/etc/origin/calico \
  -e ETCD_CA_CERT_FILE={{ calico_etcd_ca_cert_file }} \
  -e ETCD_CERT_FILE={{ calico_etcd_cert_file }} \
  -e ETCD_KEY_FILE={{ calico_etcd_key_file }} \
  -e NODENAME={{ openshift.common.hostname }} \
- -v /var/log/calico:/var/log/calico \
- -v /lib/modules:/lib/modules \
- -v /var/run/calico:/var/run/calico \
- calico/node:v1.1.0
+ -v {{ calico_log_dir }}:{{ calico_log_dir }} \
+ -v {{ calico_modules_dir }}:{{ calico_modules_dir }} \
```

Comment:
I don't think this needs to be templated. Can we leave it out in favor of simplicity?